packages/lodestar/src/api/impl/validator/util.ts

```diff
@@ -0,0 +1,22 @@
+import {IBeaconChain} from "../../../chain";
+import {ITreeStateContext} from "../../../db/api/beacon/stateContextCache";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {Epoch} from "@chainsafe/lodestar-types";
+import {assert} from "@chainsafe/lodestar-utils";
+import {computeEpochAtSlot, computeStartSlotAtEpoch} from "@chainsafe/lodestar-beacon-state-transition";
+import {processSlots} from "@chainsafe/lodestar-beacon-state-transition/lib/fast/slot";
+
+export async function safeProcessSlots
+(config: IBeaconConfig, epoch: Epoch, chain: IBeaconChain): Promise<ITreeStateContext> {
+  const {epochCtx, state} = await chain.getHeadStateContext();
+  assert.lte(
+    epoch,
+    computeEpochAtSlot(config, state.slot) + 2,
```

Comment:
I don't think we need this anymore because if we are in sync, this would stall us